CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## 2.3.0-beta.1
+
+### ✨ Features and improvements
+* Add configurable fog as a root style specification ([#10564](https://github.com/mapbox/mapbox-gl-js/pull/10564))
+* Add `map.queryTerrainElevation(lngLat)` API ([#10602](https://github.com/mapbox/mapbox-gl-js/pull/10602))
+* Reuse one renderbuffer for all draping. ([#10611](https://github.com/mapbox/mapbox-gl-js/pull/10611))
```

Comment:
```suggestion
* Reduce GPU memory footprint when terrain is active by sharing a single depth stencil renderbuffer for all draping ([#10611](https://github.com/mapbox/mapbox-gl-js/pull/10611))
```

Reply:
A bit wordy so feel free to rephrase, but just want to explicitly highlight what this change does for the user (e.g. _Reducing GPU memory footprint_)